CHANGELOG.md

```diff
@@ -25,6 +26,14 @@ Details
    https://github.com/owncloud/android/issues/3060
    https://github.com/owncloud/android/pull/3437
 
+* Bugfix - Rotation triggers security lock: [#3458](https://github.com/owncloud/android/issues/3458)
```

Comment:
Since this is a fix for an issue that has not been released yet, I think that we do not need to add a calens item for it. What do you think?

Reply:
Ok, I thought about it but finally I decided to add it just to reflect the work done. As you want, I'll remove it 👍